SPECS-EXTENDED/supermin/supermin.spec

```diff
@@ -136,26 +121,16 @@ make %{?_smp_mflags}
 
 
 %install
-make DESTDIR=$RPM_BUILD_ROOT install
+%make_install
 
 mkdir -p $RPM_BUILD_ROOT%{_rpmconfigdir}/fileattrs/
 install -m 0644 %{SOURCE3} $RPM_BUILD_ROOT%{_rpmconfigdir}/fileattrs/
 install -m 0755 %{SOURCE4} $RPM_BUILD_ROOT%{_rpmconfigdir}/
 
 
-%check
```

Comment:
Why remove the check?

Reply:
Weird, I didn't intend to remove this. Looks like a mistake, I'll revert that change.